pkg/scheduler/internal/queue/events_test.go

```diff
@@ -1,9 +1,12 @@
 /*
 Copyright 2024 The Kubernetes Authors.
+
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
+
     http://www.apache.org/licenses/LICENSE-2.0
+
```

Comment:
nit: delete ws

Reply:
All the other files have those empty lines.